services/web/server/src/simcore_service_webserver/computation_comp_tasks_listening_task.py

```diff
@@ -165,12 +173,39 @@ async def listen(app: web.Application):
                 continue
 
 
+async def listening_channel_pg_engine(app: web.Application) -> Engine:
+    cfg = app[APP_CONFIG_KEY][CONFIG_SECTION_NAME]
+    pg_cfg = cfg["postgres"]
+
+    dsn = DataSourceName(
+        application_name=f"{__name__}_{id(app)}",
+        database=pg_cfg["database"],
+        user=pg_cfg["user"],
+        password=pg_cfg["password"],
+        host=pg_cfg["long_running_session_host"],
```

Comment:
so here only 1 connection will be acquired form the pool, correct?

Reply:
yes, it's a pool with only one connection anyway. but it has to stay open the whole time.